.github/workflows/_linux-test.yml

```diff
@@ -20,6 +20,11 @@ on:
         type: boolean
         default: false
         description: checks out viable/strict
+      checkout-pytorch-submodules:
+        required: false
+        default: false
+        type: string
+        description: If this job needs to checkout the submodules
```

Comment:
nit: later you use "recursive" but the phrasing of this makes it seem like `checkout-pytorch-submodules` should either be true or false. Maybe would be good to include a list of potential inputs? (false, recursive...)